src/edbee_defaults/Mudlet.tmTheme

```diff
@@ -68,7 +68,9 @@
 			<key>settings</key>
 			<dict>
 				<key>foreground</key>
-				<string>#D68B78</string>
+				<string>#000080</string>
+				<key>fontStyle</key>
+				<string> bold</string>
```

Comment:
It doesn't seem to be - bolding works. 

The online editor spits it out that way and edbee doesn't see it as a problem, so I'm leaving it be.